src/com/mapledev/FileManager.java

```diff
@@ -55,34 +63,92 @@ public static String openFile(){
     }
 
     public static void saveFile(JTextArea area){
-        JFileChooser jfc = new JFileChooser(FileSystemView.getFileSystemView().getHomeDirectory());
-        jfc.setDialogTitle("Choose destination.");
-        jfc.setFileSelectionMode(JFileChooser.FILES_AND_DIRECTORIES);
-
-        returnValue = jfc.showSaveDialog(null);
+    	if (!opened) 
+    		saveFileAs(area);
         try {
-            File f = new File(jfc.getSelectedFile().getAbsolutePath());
+            File f = new File(jfctemp.getSelectedFile().getAbsolutePath());
             FileWriter out = new FileWriter(f);
-            out.write(area.getText());
+            area.write(out);
```

Comment:
This method is supposed to save what's in the text area into a file, instead, you are writing into the Text area, why?. Are you sure this does what it's supposed to do?